packages/dynamite/dynamite/lib/src/builder/resolve_mime_type.dart

```diff
@@ -15,7 +15,7 @@ TypeResult? resolveMimeTypeDecode(
 ) {
   if (response.content != null) {
     if (response.content!.length > 1) {
-      throw Exception('Can not work with multiple mime types right now');
+      print('Can not work with multiple mime types right now. Using the first supported.');
```

Comment:
This will break since afterwards it just iterates over all entries. You need to change the entries to be just the first one.

Reply:
Why?

I'll return the first supported entry. If the first one is unsupported (like xml) the for loop will just continue to the next entry.
If no entry is returned from within the loop the loop will finish and the exception below will be thrown,

Reply:
Ahh I see, I just wasn't aware of the whole control flow. I forgot this now uses early returns and is no longer part of the monster functions :sweat_smile:. So both are actually handled correctly.